### ISSUE TITLE
Rocker rstudio at console custum user.

### ISSUE THREAD

**Issue by ssefick** [rocker-org/rocker-versioned2#684]
Hello, is there a way to run one of the rstudio based images as a user other than rstudio and as just an R console. That is the custom user has the same uid and gid on the host system -different name.

I am trying to standardize a dev container for both console users (me - crumudginly old academic), and Rstudio users.

Thanks for all of the help, and please let me know if more information is needed.

**Comment by eddelbuettel** [comment]
Have a look around at the existing [issues (open and close)](https://github.com/rocker-org/rocker-versioned2/issues) that are searchable, this has come a few times before and been discussed.

**Comment by cboettig** [comment]
@ssefick yup, this should just work, though there's a couple ways one might read this question. 

First, mapping uid and gid are crucial if you are bind-mounting volumes to a host, and these can most easily be configured for a single user with environmental variables at container creation: e.g see docs https://rocker-project.org/images/versioned/rstudio.html#environment-variables.  (Of course if you are not bind-mounting or sharing volumes across containers, uid/gid are basically irrelevant).  

Note that this extends just fine to a container with multiple users.  A typical pattern here is that a 'root' user might do something like `docker exec` to enter a running rstudio container as root and create new users on the container matching the uid, gid, and set user names and passwords (e.g. via `adduser`, note users will need a home dir to log in via RStudio).  Then those newly added users can access the container via RStudio.   

e.g.

```
docker run -d -p 8787:8787 -e PASSWORD=secretpw --name rstudio rocker/rstudio
docker exec -ti rstudio bash
```

and then add users as in any linux environment; rstudio will recognize these users.
